test/functional/ui/cursor_spec.lua

```diff
@@ -245,6 +245,17 @@ describe('ui/cursor', function()
       eq('normal', screen.mode)
     end)
 
+    -- update the highlight again to hide cursor
+    helpers.command('hi Cursor blend=100')
```

Comment:
You could test that the `mode_info` contains `blend=100`, by updating `m.attr = {background = Screen.colors.Red, blend = 100}` and rerun the `screen._mode_info` test.

Reply:
done